app/src/main/java/com/endava/myendava/activities/SignInActivity.java

```diff
@@ -46,9 +66,18 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_sign_in);
         mUnbinder = ButterKnife.bind(this);
+        setTransparentStatusBar();
         setupModule();
     }
 
+    private void setTransparentStatusBar() {
```

Comment:
All screens from the new UI will use a transparent Status Bar. Create a BaseActivity that will be extended in all other activities that will contain this method on its onCreate method, so there will not be duplicated code in all activities